src/getNumbWithRandomDigits.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+function getNumbWithRandomDigits() {
+  const DIGIT_COUNT = 4;
+  const RANDOM_SCOPE = 10;
+
+  let randomDigit = '';
+
+  while ((+randomDigit).toString().length < DIGIT_COUNT) {
```

Comment:
```suggestion
  while ((randomDigit.length < DIGIT_COUNT) {
```
It's not the same?

Reply:
No it is not. I transform the number to int, to avoid 0 as a first number)